next/components/Molecules/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,111 @@
+import ChevronLeft from '@assets/images/chevron-left.svg'
+import ChevronRight from '@assets/images/chevron-right.svg'
+import cx from 'classnames'
+import React, { useRef } from 'react'
+
+import { VerticalCardButton } from '../../HomePage/VerticalCardButton'
+
+interface SectionPromosProps {
+  listClassName?: string
+  itemClassName?: string
+  items: { element: JSX.Element | null; key: string | undefined }[]
+  shiftIndex?: number
+  visibleItems?: number
+}
+
+const Carousel = ({
+  listClassName,
+  itemClassName,
+  items,
+  shiftIndex = 1,
+  visibleItems = 3,
+}: SectionPromosProps) => {
+  const [currentItem, setCurrentItem] = React.useState(0)
+
+  const scrollerRef = useRef<HTMLUListElement>(null)
+
+  const scrollToImage = (i: number, instant = false) => {
+    setCurrentItem(i)
+    if (!scrollerRef.current) return
+    const offset = (scrollerRef.current.scrollWidth / items.length) * i
+
+    scrollerRef.current?.scroll({
+      left: offset,
+      behavior: instant ? 'auto' : 'smooth',
+    })
+  }
+
+  const nextImage = () => {
+    const nextIdx = currentItem + shiftIndex
```

Comment:
Naming conventions: We usually do not use abbreviations, so I would prefer `nextIndex`, `maxIndex`